kahuna/public/js/edits/image-editor.html

```diff
@@ -70,7 +70,7 @@
                     <gr-icon title="You need to add both a credit and description, (PNGs are also invalid).">help</gr-icon>
                 </span>
                 <div class="result-editor__info-item--png-warning"ng:if="ctrl.invalidReasons['is_invalid_png']">
-                    <strong>PNG</strong>: Pngs with transparency not supported!
+                    <strong>PNG</strong>: This type currently not supported.
```

Comment:
Maybe we could add a title tip here with some more information about what's wrong, how to fix it?